app/views/sidebar/_post_button.html.erb

```diff
@@ -1,10 +1,22 @@
+<% if params[:controller] == 'questions' %>
```

Comment:
I'd like to keep this template file simpler. Can we put this question button at the top of the questions index page?